survey/2-work.yml

```diff
@@ -31,18 +32,18 @@ questions:
   - label: Do you work on a side project?
     required: true
     choices:
-      - NO
+      - No
       - For profit
       - For learning
 
   - label: How long have you been unemployed after graduation?
     choices:
-      - 0 months
+      - 0 month
```

Comment:
A better word could be None (worked straight after graduation)

Reply:
Yeah make sense